frontend/integration-tests/views/cluster-settings.view.ts

```diff
@@ -7,7 +7,7 @@ export const heading = element(
 export const isLoaded = async () => await browser.wait(waitForNone($$('.co-m-loader')));
 export const channelUpdateLink = $('[data-test-id="current-channel-update-link"]');
 export const channelDropdownButton = $('[data-test-id="dropdown-button"]');
-export const getSelectedChannel = $$('[data-test-id="dropdown-menu"]');
+export const getSelectedChannel = $$('.pf-c-dropdown__menu-item');
```

Comment:
Going from `data-test-id` to a class selector is a step backwards :/

Reply:
changed back